test/system/src/main/java/io/pravega/test/system/framework/TestExecutorFactory.java

```diff
@@ -9,21 +9,29 @@
  */
 package io.pravega.test.system.framework;
 
+import lombok.Getter;
 import org.apache.commons.lang3.NotImplementedException;
 
 public class TestExecutorFactory {
+    @Getter(lazy = true)
     private static final TestExecutor MARATHON_SEQUENTIAL_EXECUTOR = new RemoteSequential();
 
+    @Getter(lazy = true)
```

Comment:
Using a static lazy=true getter is equivalent to a static AtomicReference. See: https://projectlombok.org/features/GetterLazy
We disallow this by convention. If these need to be static and they don't need to be lazy, we can just make them normal static final variables. If they need to be lazy they should be made non-static.

Reply:
removed static.